python/paddle/fluid/concurrency.py

```diff
@@ -141,6 +141,8 @@ def channel_send(channel, value):
         channel (Variable|Channel): Channel variable created using
         `make_channel`.
         value (Variable): Value to send to channel
+        copy (bool): Copy data while channel send. If False, then data
+        is moved. The input cannot be used after move.
```

Comment:
Please add the default value of `copy` at the end of line145.
Maybe replacing `copy` with `copy_mode` is better.

Reply:
Sorry, I just realized that I missed your comment and merged the PR. I agree with you. Let me correct it and send you the review separately.

Reply:
it doesn't matter, the current scenario for using channel is still less.